components/ProfileHeader/ProfilePicturePicker.js

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import * as ImagePicker from 'expo-image-picker';
+import Constants from 'expo-constants';
+import * as Permissions from 'expo-permissions';
+import DeprecatedViewPropTypes from 'react-native/Libraries/DeprecatedPropTypes/DeprecatedViewPropTypes';
+
+export default class ProfilePicturePicker extends React.Component {
+  state = {
+    image: null,
+    select: true, 
+  };
+
+  render() {
+    let { image } = this.state;
+    if (this.props.openCameraRoll) {
+      console.log("Picking image")
+      this._pickImage(); 
+    } /*else {
+      console.log("Taking picture")
+      this._takeImage(); 
+    }*/
+    return (
+      <></>
+    );
+  }
+
+  componentDidMount() {
+    this.getPermissionAsync();
+  }
+
+  getPermissionAsync = async () => {
+    if (Constants.platform.ios /*&& this.props.openCameraRoll*/) {

```

Comment:
@ashwin1104 or @shresta4 do we need to do a check for android here?

Reply:
Documentation - https://docs.expo.io/versions/latest/sdk/imagepicker/ - didn't have anything about it. I saw this https://stackoverflow.com/questions/57360973/expo-imagepicker-asks-for-permissions-but-nothing-else-happens?rq=1 but the response only checks for ios, so we might not have to? I can search some more about this

Reply:
"the result is on android can access to PhotoLibrary, because there is no need to ask permission," you're prob right, saw this on a website